topiary/src/atom_collection.rs

```diff
@@ -519,7 +524,7 @@ impl AtomCollection {
                                     let multiline_only =
                                         *condition == ScopeCondition::MultiLineOnly;
                                     let new_atom = if multiline == multiline_only {
-                                        Some((**atom).clone())
+                                        Some(atom.deref().clone())
```

Comment:
Out of curiosity, is this just syntactic sugar? (Not that that's a bad thing :slightly_smiling_face:)